exchanges/btcmarkets/btcmarkets.go

```diff
@@ -379,52 +536,55 @@ func (b *BTCMarkets) SendHTTPRequest(path string, result interface{}) error {
 }
 
 // SendAuthenticatedRequest sends an authenticated HTTP request
-func (b *BTCMarkets) SendAuthenticatedRequest(reqType, path string, data, result interface{}) (err error) {
+func (b *BTCMarkets) SendAuthenticatedRequest(method, path string, data interface{}, result interface{}) (err error) {
```

Comment:
This method does not factor in sending an authed GET request with query parameters (GetTradeHistory for example)